libraries/apollo-runtime/api/apollo-runtime.api

```diff
@@ -24,6 +25,7 @@ public final class com/apollographql/apollo3/ApolloCall : com/apollographql/apol
 	public synthetic fun httpMethod (Lcom/apollographql/apollo3/api/http/HttpMethod;)Ljava/lang/Object;
 	public fun ignorePartialData (Ljava/lang/Boolean;)Lcom/apollographql/apollo3/ApolloCall;
 	public synthetic fun ignorePartialData (Ljava/lang/Boolean;)Ljava/lang/Object;
+	public final fun replaceClientHttpHeaders (Ljava/lang/Boolean;)Lcom/apollographql/apollo3/ApolloCall;
```

Comment:
🚲 🏡 : `ignoreApolloClientHttpHeaders(true)` ?

```kotlin
apolloClient.query(myQuery).addHttpHeader("key", "value")
              .ignoreApolloClientHttpHeaders(true)                             
              .execute()
```

Reply:
Or just `ignoreClientHttpHeaders` if we aim for brevity

Reply:
I'm 👍  with `ignoreApolloClientHttpHeaders`

Reply:
Sorry I changed my mind 😅 I think `ignore` may be confusing, as it may be interpreted as the ApolloClient headers will be ignored even if this call doesn't have its own headers set, which is not true. WDYT?

Reply:
The same with `replaceClientHttpHeaders(true)`. If the call has no headers (headers = null), what should `replaceClientHttpHeaders(true)` do?

Feels like we should handle `call.headers = null` as `call.headers = emptyList()` and overwrite the client headers if that's the case?

Reply:
> Feels like we should handle call.headers = null as call.headers = emptyList() and overwrite the client headers if that's the case?

Yes I think it's a good idea 👍. Removes an ambiguity.

I'm ok with `ignore` then (thought of another option: `override`)

Reply:
Not a fan of `override` because it can be understood as "I can override individual header names" like the `"key"` example below